packages/svelte2tsx/src/svelte2tsx/processInstanceScriptContent.ts

```diff
@@ -244,7 +244,7 @@ export function processInstanceScriptContent(
         handleStore(node, parent);
     };
 
-    const handleIdentifier = (ident: ts.Identifier, parent: ts.Node) => {
+	const handleIdentifier = (ident: ts.Identifier, parent: ts.Node) => {
```

Comment:
Seems like this became a tab. This repo uses spaces for intendation (maybe we should align that with the main repo some day ...)

```suggestion
    const handleIdentifier = (ident: ts.Identifier, parent: ts.Node) => {
```